packages/notifications-system/src/drivers/sendEmail/index.js

```diff
@@ -0,0 +1,29 @@
+const nodemailer = require('nodemailer');
+require('dotenv').config();
+
+const prefix = require("../../../config/environment");
+
+async function main() {
+
+  // create reusable transporter object using the Gmail SMTP transport
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
+    port: 587,
+    secure: false, // true for 465, false for other ports
+    auth: {
+      user: process.env[`${prefix}EMAIL_ACCOUNT`], // email of the account that will send the emails
+      pass: process.env[`${prefix}EMAIL_PASS`], // password of the account
+    },
+  });
+
+  // send mail with defined transport object.
+  transporter.sendMail({
+    from: `Testing Booking System <${process.env.DEV_EMAIL_ACCOUNT}>`, // sender address
```

Comment:
Look the name of the env var, we are using `DEV`, if we need use this in production, what we need to do?

Reply:
I changed the variables on the .env.example and I used the prefix in all of the environment variables that required it